nixos/modules/services/networking/prosody.nix

```diff
@@ -336,10 +336,10 @@ let
         '';
       };
 
-      vcard_muc = mkOption {
-        type = types.bool;
-        default = true;
-      description = lib.mdDoc "Adds the ability to set vCard for Multi User Chat rooms";
+      extraModules = mkOption {
+        type = types.listOf types.str;
+        default = [ "muc_mam" "vcard_muc" ];
+        description = "Add more modules to the muc Component";
```

Comment:
Capitalize `MUC` to match the documentation in other settings. Also remove the unnecessary capitalization from `Component`.